test_app/settings.py

```diff
@@ -46,7 +46,7 @@
     'test_app',
 ]
 
-AUTH_USER_MODEL = 'auth.User'
+AUTH_USER_MODEL = 'test_app.User'
```

Comment:
this is defined twice in this file. That was a mistake and the earlier assignment (this) should be deleted.